react-tutorial/src/styles.css

```diff
@@ -0,0 +1,90 @@
+* {
```

Comment:
These styles should go in the component folder. For example the classes related to Board should be in /Board in a file called "styles.css". Then you should import that file from the component.js